### ISSUE TITLE
Investigate webpack 5

### ISSUE THREAD

**Issue by sublimator** [coilhq/web-monetization-projects#1005]
Keep ahead of the bitrot

Some issues with version conflicts
```
➤ YN0000: ┌ Resolution step
➤ YN0060: │ web-monetization@workspace:. provides webpack@npm:5.0.0-rc.3 [e0cd4] with version 5.0.0-rc.3 which doesn't satisfy ^2.2.0 || ^3.0.0 || ^4.0.0-alpha requested by webpack-dev-server@npm:3.0.0-alpha6
➤ YN0002: │ @webpack-cli/init@npm:1.0.1-rc.0 doesn't provide webpack-cli@3.x.x || 4.x.x requested by @webpack-cli/generators@npm:1.0.1-rc.0
➤ YN0002: │ @webpack-cli/init@npm:1.0.1-rc.0 doesn't provide webpack-cli@4.x.x || 5.x.x requested by @webpack-cli/utils@npm:1.0.1-rc.0
➤ YN0002: │ @webpack-cli/init@npm:1.0.1-rc.0 doesn't provide webpack@4.x.x || 5.x.x requested by @webpack-cli/utils@npm:1.0.1-rc.0
➤ YN0002: │ @webpack-cli/generators@npm:1.0.1-rc.0 [86b94] doesn't provide webpack@4.x.x || 5.x.x requested by @webpack-cli/utils@npm:1.0.1-rc.0
➤ YN0060: │ webpack-dev-server@npm:3.0.0-alpha6 [e0cd4] provides webpack@npm:5.0.0-rc.3 [e0cd4] with version 5.0.0-rc.3 which doesn't satisfy ^2.2.0 || ^3.0.0 || ^4.0.0-alpha || ^4.0.0-beta || ^4.0.0 requested by webpack-dev-middleware@npm:2.0.6
```

Also issues with "import {Plugin} from 'webpack'" now that webpack has it's own types.d.ts  (which doesn't seem to export Plugin )

**Comment by sublimator** [comment]
This doesn't seem ready for prime time yet, so closing this as having been investigated.
Will wait for 5 final to be released

**Comment by sublimator** [comment]
5 final has been released:
https://github.com/coilhq/web-monetization-projects/pull/1155
https://github.com/coilhq/web-monetization-projects/pull/1154

Will try again on next patch release

**Comment by sublimator** [comment]
closed in #1266
